npm/extensions.bzl

```diff
@@ -10,13 +10,12 @@ load("//npm:npm_import.bzl", "npm_import", "npm_translate_lock")
 load("//npm/private:transitive_closure.bzl", "translate_to_transitive_closure")
 
 def _extension_impl(module_ctx):
-    npm_translate_lock_name = "npm"
```

Comment:
The repository name should be parameterized like it is in WORKSPACE.

Reply:
```
npm.npm_translate_lock(
    name = "npm",
    pnpm_lock = "//:pnpm-lock.yaml",
)

use_repo(npm, "npm")
```
This way users can replace `"npm"` with whatever they want.

Reply:
The bug is that the `name` attribute for `npm_translate_lock` isn't currently being used.

Reply:
👍